packages/ui/lib/src/models/media/media_model.dart

```diff
@@ -46,6 +49,9 @@ class LMMediaModel {
     }
     if (mediaType == LMMediaType.repost) {
       return 8;
+    }
+    if (mediaType == LMMediaType.poll) {
```

Comment:
Add this check in post utils as well, In `getPostType` method

Reply:
Added